backend/__tests__/lumens.test.ts

```diff
@@ -199,15 +200,55 @@ describe("ledgers", () => {
       expect(JSON.parse(cachedLedgers as string)).toEqual([
         {
           date: "2022-1-12 00:00:00",
-          transaction_count: 80,
-          operation_count: 300,
-          sequence: 10003,
+          data: {
```

Comment:
I might be missing something, but why does each day here need to store that day's average?

couldn't we just store the total number of ledgers and total duration from first ledger to last ledger? Then in `formatOutput` it could compute the averages of all the days then.